### DISCUSSION TITLE
Numerical issue in predator-prey model

### DISCUSSION THREAD

**Discussion by tomfid** [Data4DM/BayesSD#91] [people; relating]
If the beta parameter (prey death rate) is near 0, the prey population grows without bound. This causes a problem in Vensim in the unlucky condition when beta is just close enough to 0 for the prey to approach MAX DOUBLE (about 10^307). Then an interpolation breaks in the Powell optimizer. Adjusting the prior on beta doesn't help, because the point still gets tried.

It's possible that something similar is responsible for some of the anomalous results seen with this model in Stan. I'm currently puzzled by the need to restrict the Stan version to 2+1 parameters, for example, because I increased the Vensim version to 6+1 (adding the 2 initial conditions) with no ill effects.

My temporary solution is to set a lower bound of .001 on beta - this is well outside the prior so it shouldn't matter.

**Comment by hyunjimoon** [comment]
https://discourse.mc-stan.org/t/can-i-express-that-the-lower-bound-on-a-parameter-should-be-greater-than-zero/28572 might be relevant i.e. giving untouchable lower bound (0).
